client/landing/stepper/declarative-flow/hundred-year-plan.ts

```diff
@@ -110,6 +114,12 @@ const HundredYearPlanFlow: Flow = {
 			};
 
 			switch ( _currentStep ) {
+				case 'diy-or-difm':
+					if ( 'diy' === providedDependencies?.diyOrDifmChoice ) {
+						return navigate( hasSite ? 'site-picker' : 'setup' );
```

Comment:
If user select `diy`, should we show `new-or-existing-site` as a first step instead?

Reply:
Yes, I'll change it!